ground/src/main/java/com/google/android/ground/persistence/local/room/RoomLocalDataStore.java

```diff
@@ -128,7 +128,7 @@ private Completable insertOrUpdateOption(String taskId, Option option) {
         .subscribeOn(schedulers.io());
   }
 
-  private Completable insertOrUpdateOptions(String taskId, ImmutableList<Option> options) {
+  private Completable insertOrUpdateOptions(String taskId, List<Option> options) {
```

Comment:
This and other collections should be immutable.

Should we start using kotlinx immutable collections instead of Guava.

Reply:
Hmm, I tried switching to the kotlinx.collections.immutable.ImmutableList, but I'm having a hard time figuring out how to create a Kotlin ImmutableList from a java class, and right now we're constructing a MultipleChoice from the MultipleChoiceEntity.java.

The Kotlin ImmutableList doesn't seem to have a builder and I don't think we can access the toImmutableList extension function from a Java class.

Let me know if you have any ideas to workaround this, or if we should stick to the Guava Immutable Collections for now.

Reply:
Synced offline and decided to stick with Guava Immutable Collections for now.